.travis.yml

```diff
@@ -7,6 +7,9 @@ install:
     - pip install --only-binary=numpy,scipy
     - pip install -r requirements.txt
     - pip install -r test-requirements.txt
+script: flake8
+after_success:
+    - bash <(curl -s https://codecov.io/bash)
 script:
     - python -m unittest discover tests
```

Comment:
Can you make this `coverage run -m unittest discover tests`? Otherwise code coverage information will not be collected.

Also add `coverage` to test-requirements.txt.